core/runtime/v2/shim/util_unix.go

```diff
@@ -177,3 +225,48 @@ func CanConnect(address string) bool {
 	conn.Close()
 	return true
 }
+
+func HVSockDialer(addr string, port int, timeout time.Duration) (net.Conn, error) {
```

Comment:
If we can get away with naming this HybridVsockDialer it might be preferred. Hyper-v supports [HvSocket](https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/make-integration-service#hvsocket-socket-options) (analogous to vsock, but on windows), so even though this is in a unix file it might be a bit confusing to see

Reply:
done